### ISSUE TITLE
Integration of WebAuthn authentication strategies 

### ISSUE THREAD

**Issue by yescine** [authorizerdev/authorizer#388]
Any roadmap for integrating complex WebAuthn workflow ?

**Comment by lakhansamani** [comment]
Yes, I do have couple of issues for same, just fighting with some time as I have lot on my plate at the moment.
Plan to have this by October.

**Comment by lakhansamani** [comment]
Implemented — same WebAuthn stack as #188/#370/#449/#519: registration, login, credential management, and MFA integration. See #671, #685, #686.
